hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -789,6 +794,18 @@ message SnapshotInfo {
   optional int64 dbTxSequenceNumber = 12;
  }
 
+message SnapshotDiffJobProto {
+  optional uint64 creationTime = 1;
+  optional string jobId = 2;
```

Comment:
I'm not sure if we should return internal parameters like `volumeName`, `jobId` and `totalDiffEntries`.

Reply:
I will get rid of internal parameters.

Reply:
@hemantk-12 We can wrap `SnapshotDiffJob` with another class that has only the necessary parameters and then send that object as a response from server to client. In that case, `SnapshotDiffJob` will be moved back to its original package. What do you think?

Reply:
I see we return all the parameters in other list APIs [KeyInfo](https://github.com/apache/ozone/blob/9034434285f3877246b3ebc6a7513c082931b279/hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto#L1007) and [SnapshotInfo](https://github.com/apache/ozone/blob/9034434285f3877246b3ebc6a7513c082931b279/hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto#L777). You can ignore this comment for now.

Reply:
It is OK to return all the parameters here but we should have a client side `SnapshotDiffJob` similar to [OzoneKey](https://github.com/apache/ozone/blob/9c6cd4b93e0427fc12bbedae8156c746cbe6cea1/hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKey.java#L34) and [OzoneSnapshot](https://github.com/apache/ozone/blob/9c6cd4b93e0427fc12bbedae8156c746cbe6cea1/hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneSnapshot.java#L27). And shows only information client parameters and hides the internal parameters.

Reply:
I'll make the changes.

Reply:
@hemantk-12 I've made the client changes and this is how it looks

```
bash-4.2$ ozone sh snapshot listDiff /vol1/bucket1 --all
[ {
  "volumeName" : "vol1",
  "bucketName" : "bucket1",
  "fromSnapshot" : "snap1",
  "toSnapshot" : "snap2",
  "jobStatus" : "DONE"
} ]
```